src/AriesSDK.js

```diff
@@ -23,6 +23,12 @@ const publishMessageToBus = (message) => {
 const unwrapResponse = ({ response = {} } = {}) => response;
 
 const publishMessageToBusAndWaitForResponseWithMatchingId = (topic, params = {}) => new Promise((resolve, reject) => {
+  const featureFlags = window.featureFlags ? window.featureFlags : null;
+  const topicFeatureFlag = featureFlags && featureFlags[topic] !== undefined ? featureFlags[topic] : true;
```

Comment:
Please wrap the ternary evaluation within parens for improved readability (and furthermore: please wrap each sub-clause within parens), like so: 

```js
const topicFeatureFlag = (featureFlags && (featureFlags[topic] !== undefined)) ? featureFlags[topic] : true;
```